src/components/vuestic-sass/global/typography-link-style.demo.vue

```diff
@@ -1,37 +0,0 @@
-<template>
```

Comment:
And why was this file deleted?

Reply:
I think this file is unnecessary. 
This file does not provide useful information

Reply:
@Kreezag is this still not clear?